src/main.cpp

```diff
@@ -106,6 +118,33 @@ void splitWorkspace(const std::string& workspace)
     HyprlandAPI::invokeHyprctlCommand("dispatch", "workspace " + getWorkspaceFromMonitor(getCurrentMonitor(), workspace));
 }
 
+void splitCycleWorkspaces(const std::string& value)
+{
+    int delta = getDelta(value);
+	if (delta == 0) {
+		return;
+	}
```

Comment:
Add a `Debug::log(WARN, "message")` statement here to inform the user that the command got a bad value